extensions/interactions/NumericInput/directives/numeric-input-validation.service.ts

```diff
@@ -38,6 +38,14 @@ export class NumericInputValidationService {
       private baseInteractionValidationServiceInstance:
         baseInteractionValidationService) {}
 
+  isCustomizationArgTrue(
```

Comment:
Do we need to specify what customization arg this refers to? Note that we may, in the future, have several customisation args for numeric input interactions.

Reply:
this is replaced by ui_config